src/components/menu/menu.tsx

```diff
@@ -6,15 +7,26 @@ import {
     Prop,
     Element,
     Watch,
+    State,
 } from '@stencil/core';
 import { createRandomString } from '../../util/random-string';
 import { zipObject } from 'lodash-es';
+import { LimelBreadcrumbsCustomEvent } from '@limetech/lime-elements';
+
+import { BreadcrumbsItem } from '../breadcrumbs/breadcrumbs.types';
+import { ListSeparator } from '../list/list-item.types';
+import { OpenDirection, MenuItem, SurfaceWidth } from './menu.types';
+
 import {
-    ListSeparator,
-    MenuItem,
-    OpenDirection,
-    SurfaceWidth,
-} from '../../interface';
+    ARROW_LEFT,
+    ARROW_LEFT_KEY_CODE,
+    ARROW_RIGHT,
+    ARROW_RIGHT_KEY_CODE,
+} from '../../util/keycodes';
```

Comment:
https://github.com/Lundalogik/lime-elements/assets/35954987/5532be11-822b-492d-9b3d-20cb94e67fea

There is a bug in the keyboard navigation. When navigating the menu with keys, if I click outside to close the menu, then I can't open it again by clicking on it. (I think this happens if I have used the left and right arrow keys in my keyboard navigation)

Reply:
Has this been fixed, or is this a call for help from the reviewer?

Reply:
No I just saw that this bug exists.

Reply:
Ah, I understand. You've helped Tommy and Björn so much with this I stopped thinking of you as a reviewer and started thinking about you as one of the creators of the PR 🤦‍♂️ 😅

For me, it seems like it's pretty consistent that once I've opened the menu through keyboard navigation, regardless of how I closed it (clicking outside, clicking on a (leaf node) menu item, pressing escape, or pressing enter to select a (leaf node) menu item), I can't open the menu again by clicking the menu-button. If I tab to the menu button, I have to press enter twice to open the menu. This also seems consistent for me.

I've also tried opening the menu by clicking the button, and _then_ navigating with the arrow keys, and selecting a (leaf node) menu item, and then I can still open the menu by clicking it again. So there seems to be something that is mucked up by opening the menu using the keyboard. I would guess that there is some internal state that is set only when the menu is opened using the keyboard, and that is only reset by again trying to open the menu using the keyboard (which would explain why the first enter-press does nothing).

Reply:
Further evidence: if I get into the bugged state, then tab to the button and press enter _once_, I can then open the menu by clicking on it, as usual.

Reply:
After looking into it a bit it seems like what's causing this is the `makeEnterClickable` class that is supposed to prevent holding the Enter to produce a lot of click events, that is used for `limel-button`

Sometime the Up/Down events are triggered in the opposite order i.e. First "Up", then "Down", which will tell the click handler that the button is already "active" and should not let click events passed. So when you use Enter again, the up/down is triggered in the right order that will make it work again. Not sure how to fix it though.

Reply:
I can re-create it in this docs as well: https://lundalogik.github.io/lime-elements/versions/NEXT-37.1.0-next.52/#/component/limel-menu/

Reply:
I've created a new issue for this: https://github.com/Lundalogik/lime-elements/issues/2579

Resolving this thread.